tests/h/services/oauth_validator_test.py

```diff
@@ -207,7 +207,7 @@ def test_returns_clients_redirect_uri(self, svc, client):
             client.id, OAuthRequest("ftps://example.org")
         )
 
-        assert "ftps://example.org/auth/callback" == actual
+        assert actual == "ftps://example.org/auth/callback"
```

Comment:
:+1: I really hate this as it always breaks my brain about what it's supposed to be in the test output